funnel/models/space.py

```diff
@@ -125,8 +131,8 @@ def proposals_by_confirmation(self):
         else:
             basequery = Proposal.query.filter_by(proposal_space=self)
         response = dict(
-            confirmed=basequery.filter_by(status=PROPOSALSTATUS.CONFIRMED).order_by(db.desc('created_at')).all(),
-            unconfirmed=basequery.filter(Proposal.status != PROPOSALSTATUS.CONFIRMED, Proposal.status != PROPOSALSTATUS.DRAFT).order_by(db.desc('created_at')).all())
+            confirmed=basequery.filter_by(_status=PROPOSALSTATUS.CONFIRMED).order_by(db.desc('created_at')).all(),
+            unconfirmed=basequery.filter(~Proposal._status.in_([PROPOSALSTATUS.CONFIRMED, PROPOSALSTATUS.DRAFT])).order_by(db.desc('created_at')).all())
```

Comment:
Never touch `_status` or `_state` again. Add `UNCONFIRMED` as a group that includes everything that is not `DRAFT` or `CONFIRMED`. Use `group_by` here to get everything in one query.

Reply:
@jace `group_by` is used for aggregations, right? that doesn't really apply here.